airflow/serialization/__init__.py

```diff
@@ -15,5 +15,5 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
 """DAG serialization."""
+from __future__ import annotations
```

Comment:
This file is otherwise empty, and so I think it should not have this import and stay that way.

Reply:
Yeah. I will check for those -  I only skipped these __init__.py files that were literally "licence comment only" - those that have docstring were not seen as "empty" :)

But it will be easy to filter out

Reply:
This is the only one I've seen so far.

Reply:
Fixed.